paf-mvp-core-js/src/monitoring/tracer-factory.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  trace,
+  Tracer,
+  Span,
+  SpanOptions,
+  SpanContext,
+  SpanStatus,
+  Attributes as SpanAttributes,
+  AttributeValue as SpanAttributeValue,
+  TimeInput,
+  Context,
+  Exception,
+} from '@opentelemetry/api';
+
+class NoopSpanContext implements SpanContext {
+  spanId: string;
+  traceFlags: number;
+  traceId: string;
+}
+
+class NoopSpan implements Span {
+  private _spanContext: SpanContext = new NoopSpanContext();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Please simply remove the parameters from the method and avoid using eslint exceptions, in the whole file.
Example:
```typescript
  addEvent(): this {
    return this;
  }
```

Reply:
Done